settings.gradle

```diff
@@ -34,30 +34,27 @@ if (!JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_17)) {
 }
 
 
-def gradleEnterpriseServer = "https://ge.spockframework.org"
+def develocityServer = "https://ge.spockframework.org"
 def isCiServer = System.env["CI"] || System.env["GITHUB_ACTIONS"]
 
 def accessKeysAreMissing() {
   return !(System.env['GRADLE_ENTERPRISE_ACCESS_KEY']?.trim())
 }
 
-gradleEnterprise {
+develocity {
   buildScan {
-    captureTaskInputFiles = true
     uploadInBackground = !isCiServer
 
 
     if (gradle.startParameter.buildScan || (isCiServer && accessKeysAreMissing())) {
-      termsOfServiceUrl = "https://gradle.com/terms-of-service"
+      termsOfUseUrl = "https://gradle.com/terms-of-service"
```

Comment:
You should also use the updated URL (old one still works): https://gradle.com/help/legal-terms-of-use

Reply:
```suggestion
      termsOfUseUrl = "https://gradle.com/help/legal-terms-of-use"
```